src/Export/RstExporter.php

```diff
@@ -144,8 +144,10 @@ public function exportViewHelper(ViewHelperDocumentation $viewHelperDocumentatio
         switch ($packageName) {
```

Comment:
I suggest to switch to match syntax (available since PHP8.0):

    $package = match($packageName) {
        'fluid' => 'f',
        'default' => $packageName,
    };

Reply:
The whole repository needs some love: coding styles, rector, phpstan, everything. Feel free to make a PR ;-) I thought about it but there is so much to do.

Reply:
When code is touched it can be done in a better way. So, with every little change the code gets better.